### ISSUE TITLE
Original message within the body of moderation message

### ISSUE THREAD

**Issue by Ragavan1** [justingit/dada-mail#904]
The original message send as an attachment in the "Moderation Needed" massage is not loading in the new iOS device.  I don't know its Apple's issue or DADA issue.  However, I would prefer to attach the original message as an inline message. At lease an option to control the method.

**Comment by justingit** [comment]
The entire message to moderate is already attached with a Content-Disposition of, "inline". 

https://github.com/justingit/dada-mail/blob/master/dada/plugins/bridge#L3751-L3755

How would you modify this?

**Comment by Ragavan1** [comment]
Yes, that is correct.  Since it is attached as multipart message the iOS device is showing it as a separate file and it is not downloading it properly to view the original message.  I checked it using iPhone 11 Pro Max and iPad Pro Max 12.9. with iOS 13.3.1 Version.  It may be Apple's issue.  

However, to overcome that issue, Is there any way to add variable & tag  to **moderation_msg.html** similar to the one in the mailing_list_message-discussion.html  so that the body of the original message will be in the body of the moderation message.  Currently, it is there as an attachment.

Sorry, I am not an IT Guy.  I am a self-learning individual therefore the terms that I used here may not be accurate.

**Comment by justingit** [comment]
> Is there any way to add variable & tag to moderation_msg.html similar to the one in the mailing_list_message-discussion.html so that the body of the original message will be in the body of the moderation message.

No. The message to be moderated will probably be a multipart message itself (HTML version, plain text version, attached images + files). 

There's also no variable in mailing_list_message-discussion.html to add the body of the original message that's being replied to.  It's the mail reader that would be quoted the original message, when a user replies to the message. But data is lost, and the original message is munged.

**Comment by Ragavan1** [comment]
> > Is there any way to add variable & tag to moderation_msg.html similar to the one in the mailing_list_message-discussion.html so that the body of the original message will be in the body of the moderation message.
> 
> No. The message to be moderated will probably be a multipart message itself (HTML version, plain text version, attached images + files).
> 
> There's also no variable in mailing_list_message-discussion.html to add the body of the original message that's being replied to. It's the mail reader that would be quoted the original message, when a user replies to the message. But data is lost, and the original message is munged.

So how it is working on the "mailing_list_message-discussion.html"  Because, that is the ultimate email goes to all the subscribers.

**Comment by justingit** [comment]
What you're asking for is, "Hey why don't you just take some of the content from the original message to be moderated, and plunk it into the main message that's asking you to moderate the original message?"

I get that. What I think that hand waves is that you're _not_ now moderating the original message; you're moderating a _excerpt_ of the message and that may be missing some important parts of the original message, such as an attachment you may/may not want to be sent with the message. 
Any way you slice it, however you present the excerpt (or whatever you'd like to call it) is going to lose some of the original data of the message you want to moderate.
